Source/towners.cpp

```diff
@@ -76,7 +77,7 @@ void InitSmith(Towner &towner, const TownerData &townerData)
 	towner.animOrder = AnimOrder;
 	towner.animOrderSize = sizeof(AnimOrder);
 	LoadTownerAnimations(towner, "Towners\\Smith\\SmithN.CEL", 16, 3);
-	towner.name = _("Griswold the Blacksmith");
+	towner.name = _("Griswold the Penis");

```

Comment:
```suggestion
	towner.name = _("Griswold the Blacksmith");
```

Reply:
This wasn't meant to make it here! Big mb